### PR TITLE
[IMP] mail: rephrase the demo data

### PR DESCRIPTION
**Pull request by zel-odoo**
The demo data for the Discuss sub-thread has been rephrased to be less wordy. And also fix the problem that the single quote is not correctly used.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by alexkuhn — APPROVED** [review]
@robodoo delegate+

**Comment by zel-odoo** [comment]
@robodoo r+

File: addons/mail/demo/discuss_channel_demo.xml [hunk 1]
```diff
@@ -198,27 +198,26 @@
             <field name="message_type">comment</field>
             <field name="subtype_id" ref="mail.mt_comment"/>
             <field name="author_id" ref="base.partner_demo"/>
-            <field name="body"><![CDATA[<p>I can’t seem to access the team
-            calendar. It keeps saying ‘access denied.’ Has anyone else run into
-            this?</p>]]></field>
+            <field name="body"><![CDATA[<p>I can't access the team calendar.
```
Comment (tde-banana-odoo) [inline addons/mail/demo/discuss_channel_demo.xml:201]: Using CDATA in 2024 :o

Comment (tde-banana-odoo) [inline addons/mail/demo/discuss_channel_demo.xml:201]: Time to use real html type for the field ^^

**Pull request closed without merging**